### PR TITLE
Clinic schema

### PR DESCRIPTION
**Pull request by akucintavalent**
In this pull request, we created a file named `schema_based_on_diagram.sql` where we implemented the database from the diagram.

image

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  FOREIGN KEY (treatment_id) REFERENCES treatments(id) ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
+CREATE TABLE medical_treatments(
+  medical_history__id INT NOT NULL,
+  treatment_id INT NOT NULL,
+  FOREIGN KEY (medical_history__id) REFERENCES medical_histories(id) ON DELETE RESTRICT ON UPDATE CASCADE,
+  FOREIGN KEY (treatment_id) REFERENCES treatments(id) ON DELETE RESTRICT ON UPDATE CASCADE,
+  PRIMARY KEY (medical_history__id, treatment_id)
+);
```
Comment (mmsesay) [inline schema_based_on_diagram.sql:49]: - [x] Good job creating these tables and their relationships. However, I believe you forgot to include the indexes for the foreign keys. I'm suggesting you add the indexes for all the foreign keys.

See Examples:

```
CREATE INDEX ON medical_treatments (treatment_id);
CREATE INDEX ON medical_treatments (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
```

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (AmaduKamara) [reply schema_based_on_diagram.sql:49]: Ooh, yeah. Thanks, bro.

Comment (AmaduKamara) [reply schema_based_on_diagram.sql:49]: Added indexes for all Foreign Keys

**Review by mmsesay — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**